flink-connector-examples/build.gradle

```diff
@@ -24,7 +24,7 @@ archivesBaseName = 'pravega-flink-examples'
 dependencies {
     compile "de.javakaffee:kryo-serializers:${kryoSerializerVersion}"
     compile "org.scala-lang.modules:scala-java8-compat_$flinkScalaVersion:${scalaJava8CompatVersion}"
-    compile "io.pravega:pravega-connectors-flink_$flinkScalaVersion:${flinkConnectorVersion}"
+    compile "io.pravega:pravega-connectors-flink-${flinkMajorMinorVersion}_$flinkScalaVersion:${flinkConnectorVersion}"
```

Comment:
Seems awkward to use `${flinkMajorMinorVersion}` vs `$flinkMajorMinorVersion`.  Choose one:
```
"io.pravega:pravega-connectors-flink-$flinkMajorMinorVersion_$flinkScalaVersion:$flinkConnectorVersion"
```
or
```
"io.pravega:pravega-connectors-flink-${flinkMajorMinorVersion}_${flinkScalaVersion}:${flinkConnectorVersion}"
```

Reply:
Fixed all with `{}`